arbos/parse_l2.go

```diff
@@ -96,6 +97,7 @@ const (
 	L2MessageKind_Heartbeat          = 6 // deprecated
 	L2MessageKind_SignedCompressedTx = 7
 	// 8 is reserved for BLS signed batch
+	L2MessageKind_EspressoTx = 10
```

Comment:
This is a new transaction type that indicates that the tx that follows is a tx coming from the espresso sequencer. The handling of espresso transactions by the sequencer is different than for the normal transaction in that the sequencer will not filter out malformed or otherwise invalid transactions (e. g. bad nonce).